test/packaging/ansible/test.yml

```diff
@@ -1,29 +1,29 @@
 ---
```

Comment:
I think that this should be uncommented 👇

Reply:
done